mlrun/__main__.py

```diff
@@ -994,6 +994,13 @@ def project(
                 timeout=timeout,
                 overwrite=overwrite_schedule,
             )
+        except mlrun.errors.MLRunConflictError as error:
+            # We don't want to print traceback in this case because it will add the SDK error with "overwrite = True"
+            message = f"failed to run pipeline, {error}".replace(
```

Comment:
I would check if the error message you except is related to the `overwrite` parameter and if it does, just raise a new exception with the `--overwrite-schedule` and won't do anything related to the `had_error` or prints and lets the latter exception handling block do all that for you